website/content/docs/commands/var/lock.mdx

```diff
@@ -0,0 +1,109 @@
+
+## Restrictions
+
+Variable paths are restricted to [RFC3986][] URL-safe characters that don't
+conflict with the use of the characters `@` and `.` in template blocks. This
+includes alphanumeric characters and the special characters `-`, `_`, `~`, and
+`/`. Paths may be up to 128 bytes long. The following regex matches the allowed
+paths: `^[a-zA-Z0-9-_~/]{1,128}$`
+
+The keys for the items in a variable may contain any character, but keys
+containing characters outside the set of Unicode letters, Unicode digits, and
+the underscore (`_`) can not be read directly using dotted references in Nomad's
+template engine. Instead, they require the use of the `index` template function
+to directly access their values. This does not impact cases where the keys and
+values are read using the `range` function.
+
+Variable items are restricted to 64KiB in size. This limit is calculated by
+taking the sum of the length in bytes of all of the unencrypted keys and values.
+
+## General Options
+
+@include 'general_options.mdx'
+
+## Lock Options
+- `-verbose`: Provides additional information via standard error to preserve
+  standard output (stdout) for redirected output.
+
+- `ttl`: Optional, TTL for the lock, time the variable will be locked. Defaults to 15s.
+
+- `delay`: Optional, time the variable is blocked from locking when a lease is not renewed.	
+	Defaults to 15s.
+
+- `max-retry`:Optional, max-retry up to this number of times if Nomad returns a 500 error
+	while monitoring the lock. This allows riding out brief periods of
+	unavailability without causing leader elections, but increases the amount of
+	time required to detect a lost lock in some cases. Defaults to 5. Set to 0 to
+	disable.
+
+- `shell`: Optional, use a shell to run the command (can set a custom shell via		
+	the SHELL environment variable). The default value is true.
+
+## Examples
+
+Attempst to acquire a lock over the variable at path "secret/creds" for a time of 
+15s and executes  `nomad job run webapp.nomad.hcl` if it succeds:
+
+```shell-session
+$ nomad var lock -ttl=15s secret/creds "nomad job run webapp.nomad.hcl"
+```
+
+The data can also be consumed from a file on disk by prefixing with the "@"
+symbol. For example, you can store a variable using a specification created with
+the `nomad var init` command.
+
+```shell-session
+$ nomad var lock secret/foo @spec.nv.json `nomad job run webapp.nomad.hcl`
+```
+
+## SHELL
+
+Consul lock launches its children in a shell. By default, Consul will use the shell
+defined in the environment variable `SHELL`. If `SHELL` is not defined, it will
+default to `/bin/sh`. It should be noted that not all shells terminate child
+processes when they receive `SIGTERM`. Under Ubuntu, `/bin/sh` is linked to `dash`,
+which does **not** terminate its children. In order to ensure that child processes
+are killed when the lock is lost, be sure to set the `SHELL` environment variable
+appropriately, or run without a shell by setting `-shell=false`.
```

Comment:
I think this might be duplicate content that you already have above

Reply:
Removed!